hudi-client/src/main/java/org/apache/hudi/index/hbase/HBaseIndex.java

```diff
@@ -182,6 +182,7 @@ private boolean checkIfValidCommit(HoodieTableMetaClient metaClient, String comm
     // 2) is less than the first commit ts in the timeline
     return !commitTimeline.empty()
         && (commitTimeline.containsInstant(new HoodieInstant(false, HoodieTimeline.COMMIT_ACTION, commitTs))
```

Comment:
we could use the `metaClient.getCommitsTimeline()` or one of those methods which will automatically determine the commit instant type based on table type?

Reply:
Yes, @rmpifer please use the above API Vinoth mentioned to refactor this code so we can include both DELTA_COMMIT & COMMIT actions

Reply:
Thanks for the suggestion. Updated to address comment